unit_test/test_spdm_responder/version.c

```diff
@@ -223,6 +230,8 @@ void libspdm_test_responder_version_case6(void **state)
     assert_int_equal(spdm_response->header.param1,
```

Comment:
This should probably be a `VersionMismatch` error response, but I'll file a separate issue for that.